test-assets/ntp-release/config/dev.yml

```diff
@@ -1,2 +1,3 @@
 ---
-latest_release_filename: "/root/hcf/test_releases/ntp-release/dev_releases/ntp/ntp-2+dev.2.yml"
+latest_release_filename: "/home/ericp/lab/gostuff/src/github.com/hpcloud/fissile/test-assets/ntp-release/dev_releases/ntp/ntp-2+dev.4.yml"
```

Comment:
Agreed. Committed in 115ae0e . Note that these errors are common (path is localized when you
recreate a bosh release) but inconsequential, as the tests don't use this value.